forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -383,3 +390,51 @@ def post():
         except BusinessException as err:
             current_app.logger.warning(err.error)
             return err.error, err.status_code
+
+@API.route("/<string:form_id>/submission/<string:submission_id>/render", doc=False)
+class FormResourceRenderFormPdf(Resource):
+    """Resource to render form and submission details as html for FormResourceExportFormPdf function """
+
+    @staticmethod
+    @auth.require
+    @profiletime
+    def get(form_id: string, submission_id: string):
+        formio_service = FormioService()
+        form_io_token = formio_service.get_formio_access_token()
+        req_data = {'form_id' : form_id, 'sub_id': submission_id }
+
+        form_data = formio_service.get_form(req_data, form_io_token)
+        submission_data = formio_service.get_submission(req_data, form_io_token)
+        form_info = {'form_data': form_data, 'submission_data' : submission_data}
```

Comment:
Once the form adapter is added @auslin-aot have a update in get the submission here